core/dist/css/decanter-grid.css

```diff
@@ -674,498 +674,1590 @@ body {
+  .flex-sm-10-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(83.3333333333% - 6px);
             flex: 0 0 calc(83.3333333333% - 6px);
     max-width: calc(83.3333333333% - 6px); } }
 
-@media only screen and (min-width: 1200px) and (max-width: 1499px) {
-  .flex-xl-10-of-12 {
+@media only screen and (min-width: 1200px) {
+  .flex-sm-10-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(83.3333333333% - 6.6666666667px);
             flex: 0 0 calc(83.3333333333% - 6.6666666667px);
     max-width: calc(83.3333333333% - 6.6666666667px); } }
 
 @media only screen and (min-width: 1500px) {
-  .flex-2xl-10-of-12 {
+  .flex-sm-10-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(83.3333333333% - 8px);
             flex: 0 0 calc(83.3333333333% - 8px);
     max-width: calc(83.3333333333% - 8px); } }
 
-@media only screen and (min-width: 0) and (max-width: 575px) {
-  .flex-xs-11-of-12 {
-    -webkit-box-flex: 0;
-        -ms-flex: 0 0 calc(91.6666666667% - 1.6666666667px);
-            flex: 0 0 calc(91.6666666667% - 1.6666666667px);
-    max-width: calc(91.6666666667% - 1.6666666667px); } }
-
-@media only screen and (min-width: 576px) and (max-width: 767px) {
+@media only screen and (min-width: 576px) {
   .flex-sm-11-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(91.6666666667% - 1.6666666667px);
             flex: 0 0 calc(91.6666666667% - 1.6666666667px);
     max-width: calc(91.6666666667% - 1.6666666667px); } }
 
-@media only screen and (min-width: 768px) and (max-width: 991px) {
-  .flex-md-11-of-12 {
+@media only screen and (min-width: 768px) {
+  .flex-sm-11-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(91.6666666667% - 1.6666666667px);
             flex: 0 0 calc(91.6666666667% - 1.6666666667px);
     max-width: calc(91.6666666667% - 1.6666666667px); } }
 
-@media only screen and (min-width: 992px) and (max-width: 1199px) {
-  .flex-lg-11-of-12 {
+@media only screen and (min-width: 992px) {
+  .flex-sm-11-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(91.6666666667% - 3px);
             flex: 0 0 calc(91.6666666667% - 3px);
     max-width: calc(91.6666666667% - 3px); } }
 
-@media only screen and (min-width: 1200px) and (max-width: 1499px) {
-  .flex-xl-11-of-12 {
+@media only screen and (min-width: 1200px) {
+  .flex-sm-11-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(91.6666666667% - 3.3333333333px);
             flex: 0 0 calc(91.6666666667% - 3.3333333333px);
     max-width: calc(91.6666666667% - 3.3333333333px); } }
 
 @media only screen and (min-width: 1500px) {
-  .flex-2xl-11-of-12 {
+  .flex-sm-11-of-12 {
     -webkit-box-flex: 0;
         -ms-flex: 0 0 calc(91.6666666667% - 4px);
             flex: 0 0 calc(91.6666666667% - 4px);
     max-width: calc(91.6666666667% - 4px); } }
 
-@media only screen and (min-width: 0) and (max-width: 575px) {
-  .flex-xs-12-of-12 {
+@media only screen and (min-width: 576px) {
+  .flex-sm-12-of-12 {
```

Comment:
Is it possible to do this optimization - for all the `.flex-xx-12-of-12` classes, only write the first media query `@media only screen and (min-width: 576px)` since all the subsequent media queries generate the same numbers?
![Screen Shot 2019-06-12 at 1 55 17 AM](https://user-images.githubusercontent.com/42749717/59337922-4a4ce280-8cb6-11e9-8e4b-0224e9eccf89.png)